### ISSUE TITLE
Prediction mask are all False 

### ISSUE THREAD

**Issue by MISSEY** [facebookresearch/detectron2#2794]
Hi 

I trained my custom model using following configuration : 

```
cfg = get_cfg()
cfg.merge_from_file(model_zoo.get_config_file("COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml"))
cfg.DATASETS.TRAIN = ("street_train_dataset",)
cfg.DATASETS.TEST = ("street_val_dataset",)
cfg.TEST.EVAL_PERIOD = 1000
cfg.MODEL.WEIGHTS = model_zoo.get_checkpoint_url(
    "COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml")  # Let training initialize from model zoo
cfg.SOLVER.IMS_PER_BATCH = 2
cfg.SOLVER.BASE_LR = 0.0001  # pick a good LR
cfg.SOLVER.MAX_ITER = args.epochs
cfg.MODEL.ROI_HEADS.BATCH_SIZE_PER_IMAGE = 64  # faster, and good enough for this toy dataset (default: 512)
cfg.MODEL.ROI_HEADS.NUM_CLASSES = 3
cfg.INPUT.MIN_SIZE_TRAIN = (800,)
cfg.DATALOADER.SAMPLER_TRAIN = 'RepeatFactorTrainingSampler'

cfg.OUTPUT_DIR = settings.check_point_ouput_directory
trainer = DT(cfg)
trainer.resume_or_load(resume=True)
trainer.train()
```

Training completed successfully 

During inference all the values in pred_masks are False, I have checked manually each item in outputs["instances"].pred_masks

```
{'instances': Instances(num_instances=2, image_height=800, image_width=800, fields=[pred_boxes: Boxes(tensor([[474.3644, 713.4433, 748.6342, 729.5402],
         [393.4296, 482.9530, 468.2175, 672.8376]], device='cuda:0')), scores: tensor([0.7000, 0.6536], device='cuda:0'), pred_classes: tensor([1, 2], device='cuda:0'), pred_masks: tensor([[[False, False, False,  ..., False, False, False],
          [False, False, False,  ..., False, False, False],
          [False, False, False,  ..., False, False, False],
          ...,
          [False, False, False,  ..., False, False, False],
          [False, False, False,  ..., False, False, False],
          [False, False, False,  ..., False, False, False]],
 
         [[False, False, False,  ..., False, False, False],
          [False, False, False,  ..., False, False, False],
          [False, False, False,  ..., False, False, False],
          ...,
          [False, False, False,  ..., False, False, False],
          [False, False, False,  ..., False, False, False],
          [False, False, False,  ..., False, False, False]]], device='cuda:0')])}
```

I am getting the prediction boxes, but not the mask, what could be the possible reason ??

**Comment by ppwwyyxx** [comment]
As the [issue template](https://github.com/facebookresearch/detectron2/blob/master/.github/ISSUE_TEMPLATE/unexpected-problems-bugs.md) mentions:

> If you expect the model to converge / work better, note that we do not give suggestions on how to train a new model. Only in one of the two conditions we will help with it: (1) You're unable to reproduce the results in detectron2 model zoo. (2) It indicates a detectron2 bug.

We provide configs & models with standard academic settings and expect users to have the knowledge to choose or design appropriate models & parameters for their own tasks.
